service_container/autowiring.rst

```diff
@@ -629,7 +629,7 @@ and even :doc:`complex expressions </service_container/expression_language>`::
             bool $debugMode,
 
             // and expressions
-            #[Autowire(expression: 'service("App\\Mail\\MailerConfiguration").getMailerMethod()')]
+            #[Autowire(expression: 'service("App\\\\Mail\\\\MailerConfiguration").getMailerMethod()')]
```

Comment:
base don my experience, 3 slashes should be enough, can you please double check? Thanks

Reply:
Here is a part of my code:
<img width="831" alt="CleanShot 2023-02-11 at 13 27 10@2x" src="https://user-images.githubusercontent.com/995707/218257916-33a93fc2-079a-4197-959b-5c966aa1b6fe.png">

Reply:
@OskarStark Yes, 3 backslashes seem to be sufficient. Thank you for the hint!